PhpUnit/AbstractExtensionTestCase.php

```diff
@@ -44,13 +45,22 @@ protected function setUp()
      * Call this method from within your test after you have (optionally) modified the ContainerBuilder for this test
      * ($this->container).
      *
-     * @param array $specificConfiguration
+     * If your extension(s) implements \Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface, you may
+     * set $withPrependInvocation to TRUE to invoke prepend() method prior to load() method of your extension.
+     *
+     * @param array $configurationValues
+     * @param bool $withPrependInvocation
      */
-    protected function load(array $configurationValues = array())
+    protected function load(array $configurationValues = array(), $withPrependInvocation = false)
```

Comment:
@matthiasnoback IMO as that functionality is run automatically by Symfony DI if that interface is used, we should probably do same & not force to enable this manually, WDYT?